### DISCUSSION TITLE
Extend IDS RAM / Business Layer

### DISCUSSION THREAD

**Discussion by anilturkmayali** [International-Data-Spaces-Association/Personal-Data-and-IDS#9] [Ideas]
**What could be elaborated?** 
The integration of personal data considerations into business models, value propositions, and stakeholder interactions within IDS.

**Action Steps:**
- Analyze current business models (use cases) to identify potential gaps related to personal data handling.
- Propose business strategies that prioritize user-centricity and data sovereignty.
- Develop guidelines for businesses to engage with stakeholders on personal data matters.

**Comment by mabuse68** [comment]
On the "Personal Data Monetisation" topic.
Even if this is not the foremost concern to work out, I'd like to pinpoint some aspects that a standard data space could address:
- Identify personal data ownership: how a data space actor can be sure that personal data is effectively ownership of the subject.
- A personal, unidentifiable ID: we want to de-couple identity for monetisation purposes, yet allow tracking for law enforcement. I know that VCs are made for that, we should say how we can standardise their use for personal data monetisation and allow a lawful transaction.
- How a person can make her data discoverable. Again, the means are there, we need to tell which way is the standard for IDSA
- Disclaimer contracts. Personal monetisation could incur in taxation, fees, terms of use, right of transfer, immutability, etc. We can't assume a person has the same technical means to interpret an ODRL-extended contract. We need to define a role of a "Disclaimer translator" that clearly communicates agreement terms to people at scale (and multi-lingual).

Last, but not least: the role of brokers in the RAM might need to be adapted to take in account confidentiality of personal transactions.
